src/JitInterface/src/CorInfoHelpFunc.cs

```diff
@@ -253,9 +239,7 @@ public enum CorInfoHelpFunc
 
         // These helpers are required for MDIL backward compatibility only. They are not used by current JITed code.
         CORINFO_HELP_TYPEHANDLE_TO_RUNTIMETYPEHANDLE_OBSOLETE, // Convert from a TypeHandle (native structure pointer) to RuntimeTypeHandle at run-time
-    #if RYUJIT_CTPBUILD
         CORINFO_HELP_METHODDESC_TO_RUNTIMEMETHODHANDLE_MAYBENULL_OBSOLETE, // Convert from a MethodDesc (native structure pointer) to RuntimeMethodHandle at run-time
```

Comment:
CORINFO_HELP_METHODDESC_TO_RUNTIMEMETHODHANDLE_MAYBENULL_OBSOLETE needs to be deleted instead (the #ifdef around it is other way around compared to others). I suspect that it is the reason why the Windows tests are failing in the CI